tests/types/test_spatial.py

```diff
@@ -132,6 +133,85 @@ def test_geometry_transformer_datum_uri() -> None:
     assert geometry.transformer_datum_uri == vocab(graph=rdflib.Graph()).get(settings.Settings().DEFAULT_TARGET_CRS)
 
 
+@pytest.fixture
+def temp_default_crs(mocker: pytest_mock.MockerFixture) -> Iterator[Callable[[str], None]]:
+    """Provides a temporary value for Default CRS when called.
+
+    Args:
+        mocker: The mocker fixture
+
+    Yields:
+        Function to perform the change with new value as arg.
+    """
+    # Retain the original setting
+    original = settings.SETTINGS.DEFAULT_TARGET_CRS
+
+    # Define callable
+    def change_crs(value: str) -> None:
+        """Performs the change.
+
+        Args:
+            value: New default CRS name to use for the test.
+        """
+
+        # Create a stubbed settings model
+        class TempSettings(settings.Settings):
+            # Modified fields below
+            DEFAULT_TARGET_CRS: str = value
+
+        # Patch Settings
+        mocker.patch(
+            "abis_mapping.settings.Settings",
+            new=TempSettings,
+        )
```

Comment:
If you change spatial.py to always look up `settings.SETTINGS.DEFAULT_TARGET_CRS` (not `settings.Settings()`),
this part of the mocking could be removed.
I'll leave this up to you.

Reply:
leave